src/ui/Analytics/Analytics.ts

```diff
@@ -199,6 +225,9 @@ export class Analytics extends Component {
 
     this.bind.onRootElement(QueryEvents.buildingQuery, (data: IBuildingQueryEventArgs) => this.handleBuildingQuery(data));
     this.bind.onRootElement(QueryEvents.queryError, (data: IQueryErrorEventArgs) => this.handleQueryError(data));
+    if (this.isGtmScriptPresent()) {
```

Comment:
As discussed live, I think this should be dictated solely by the `pushToGtmDataLayer` option.

I'd keep this logic that tries to determine the presence or absence of google tag manager to determine what the "default" value for the option should be, in the case that it's left undefined.